go.mod

```diff
@@ -3,12 +3,13 @@ module github.com/grafana/grizzly
 go 1.13
 
 require (
+	github.com/cortexproject/cortex v1.2.0
```

Comment:
9MB -> 12MB.

That's still smaller than Tanka and gives us a lot of possibilities. Keep in mind Go only builds into the binary what it needs, not what is in go.mod. This also means this might increase a bit once we start using the Cortex API client